pymc3/model_graph.py

```diff
@@ -13,19 +13,19 @@
 #   limitations under the License.
 
 from collections import deque
-from typing import Dict, Iterator, Optional, Set
+from typing import Dict, Iterator, NewType, Optional, Set
 
-VarName = str
-
-from theano.compile import SharedVariable
-from theano.graph.basic import walk
-from theano.tensor import Tensor
+from aesara.compile import SharedVariable
+from aesara.graph.basic import walk
+from aesara.tensor.var import TensorVariable
 
 import pymc3 as pm
 
 from pymc3.model import ObservedRV
 from pymc3.util import get_default_varnames, get_var_name
 
+VarName = NewType("VarName", str)
```

Comment:
What's the background behind this change?

Reply:
That's the correct way to create a distinct type from an existing type (see [here](https://mypy.readthedocs.io/en/stable/more_types.html#newtypes)).  Setting `VarName = str` does/offers nothing in terms of type checking.